clang/include/clang/CIR/Dialect/IR/CIRTypes.td

```diff
@@ -53,17 +53,21 @@ def CIR_IntType : CIR_Type<"Int", "int",
     std::string getAlias() const {
       return (isSigned() ? 's' : 'u') + std::to_string(getWidth()) + 'i';
     };
+    /// Return true if this is a standard integer type (i.e. not a bit-precise
+    /// integer type).
+    bool isStandard() const {
+      return isValidStdIntBitwidth(getWidth());
```

Comment:
`isValidPrimitiveIntBitwidth`

Reply:
also please update all comments that say "standard"

Reply:
Updated.